wd_helper.au3

```diff
@@ -1471,7 +1471,7 @@ EndFunc   ;==>_WD_GetBrowserVersion
 ; ===============================================================================================================================
 Func _WD_GetWebDriverVersion($sInstallDir, $sDriverEXE)
 	Local Const $sFuncName = "_WD_GetWebDriverVersion"
-	Local $sDriverVersion = "None"
+	Local $sDriverVersion = 0
```

Comment:
This is a script breaking change. I think some additional thought should go into this before proceeding.

Reply:
Also doesn't make sense to assign a numeric value to a string variable. 😛

Reply:
There was inconsistence with Function Header 
`Failure - "" (empty string) and sets @error to one of the following values:`
It was not true because at the top of the function it was declared as `$sDriverVersion = "None"`

Reply:
`_WD_GetWebDriverVersion()` was introduced not so long ago, and is used mainly in `_WD_Update()` 
Even if it is used by any coder/scripter should be checked with `_VersionCompare()` if it was'nt checked this way it was checked wrongly, so this is not ScriptBreaking Change, it is a fix.

Reply:
That fine. But it doesn't address my initial concerns posted above.

Reply:
> Also doesn't make sense to assign a numeric value to a string variable. 😛

changed to `$sDriverVersion = "0"`

Reply:
> I think some additional thought should go into this before proceeding.

I will wait for your thoughts ;)

Reply:
Just checked:
even if we back to `Local $sDriverVersion = "None"` then `_WD_Update()` with using `_VersionCompare()` still works well.

Reply:
I'm ok with changing it to "0", "0.00", or whatever. We just need to clearly document the change as potentially script breaking just in case someone is checking for "none".

Reply:
> I'm ok with changing it to "0", "0.00", or whatever.

Let stick with 0



> We just need to clearly document the change as potentially script breaking just in case someone is checking for "none".

in REVISIONS.md ?

Reply:
Yes. I have already started the revised list locally, so I'll add it there.